apiary.apib

```diff
@@ -5,6 +5,148 @@ HOST: https://operationcode.org/
 
 API endpoints that Operation Code's Rails backend makes available to its React frontend.
 
+## Users | Create [/api/v1/users{?first_name,last_name,email,zip,password,mentor}]
+
++ Parameters
+
+  + first_name (string, required) - First Name of the user in the form of an string
+  + last_name (string, required) - Last Name of the user in the form of an string
+  + email (string, required) - Email of the User in the form of a string
+  + zip (string, required) - Zipcode of the User in the form of a string
+  + password (string, required) - Password of the User in the form on a string
+  + mentor (boolean, required) - Is User requesting mentor in the form of a boolean
```

Comment:
I believe `mentor` is optional

Reply:
I think you're [right]( https://github.com/OperationCode/operationcode_frontend/blob/bff605dc75ffeca42434d168145c40c89cceeb96/src/scenes/home/signup/signup.js#L72) from a front end perspective.